src/octree/node.rs

```diff
@@ -98,7 +107,7 @@ impl NodeId {
     }
 
     /// Returns the NodeId for the corresponding 'child_index'.
-    fn get_child_id(&self, child_index: &ChildIndex) -> Self {
+    pub fn get_child_id(&self, child_index: &ChildIndex) -> Self {
```

Comment:
nit: ChildIndex is copy and only 1 byte, so taking it by value is cheaper than taking a reference.